### DISCUSSION TITLE
Ignore tags when associated with pre-release on GitHub [terraform]

### DISCUSSION THREAD

**Discussion by ekristen** [renovatebot/renovate#11491] [Request Help]
Is there a way to configure renovatebot to ignore tags when and if they are associated with a release that's marked as pre-release?

My use case is specifically terraform, for example this terraform module sources a git repository and tag, but 0.26.x are all marked as pre-release but renovate still wants me to update.

```
module "vpc" {
  source     = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=0.25.0"
  cidr_block = var.cidr
}
```

Thank you for your time.

**Comment by viceice** [comment]
What's your renovate config. Do you override the datasource?

I think it should use the `github-releases` which should ignore prereleases.

Can you check the `packages with updates` debug log message and paste the relevant parts here.

**Reply by ekristen** [reply]
Default config and using renovate integration on GitHub. I’ll go find the logs on the app website now and follow up.

**Reply by viceice** [reply]
I've created a minimal repro https://github.com/renovate-reproductions/terraform-github-modules

```
DEBUG: packageFiles with updates (repository=renovate-reproductions/terraform-github-modules)
       "config": {
         "terraform": [
           {
             "packageFile": "main.tf",
             "deps": [
               {
                 "lookupName": "cloudposse/terraform-aws-vpc",
                 "depType": "module",
                 "depName": "github.com/cloudposse/terraform-aws-vpc",
                 "currentValue": "0.25.0",
                 "datasource": "github-tags",
                 "depIndex": 0,
                 "updates": [],
                 "warnings": [],
                 "versioning": "hashicorp",
                 "sourceUrl": "https://github.com/cloudposse/terraform-aws-vpc",
                 "currentVersion": "0.25.0",
                 "fixedVersion": "0.25.0"
               }
             ]
           }
         ]
       }
```

I don't see the v0.26.0 update

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/blob/9f791552d32d8efa5fa22e7b49d5a292ba16d72a/lib/datasource/github-tags/index.ts#L183-L203

So it can only be some wrong configuration on your side

**Reply by ekristen** [reply]
I have the standard config. I wonder if they initially weren’t marked as pre release. I’ll take a look at my logs today.  Thanks for the reproduce. I’ve got another terraform issue I have to open around the lock file today. Something is not working right with that new feature.
